### DISCUSSION TITLE
How to contribute?

### DISCUSSION THREAD

**Discussion by kikutano** [thomhurst/TUnit#606] [Q&A] [answered]
Hello to everyone, I am new to the open source environment, but I am really passionate about Automated Testing with xUnit! This TUnit is really impressive and I want to contribute by writing some code. I forked the project and tried to compile, but I've no idea where to start! There is no documentation, no comments or other things that allows you to start to play with the project's code.

So, I'm a completely newbie or someone want to start to create some tutorial or documentation for the source code? 😅

**Comment by thomhurst** [comment]
There's a documentation link on the readme and a docs folder in the repo

**Comment by kikutano** [comment]
But that documentation is to use the library not to understand how the project's code is structured and how to modify it to add features 🤔 . I'm right?

**Comment by thomhurst** [comment, marked as answer]
Oh right. I haven't written up any contribution documentation yet as the project itself is still in pre-release so I'm focusing on getting it feature completed first
